rs/sns/governance/src/proposal.rs

```diff
@@ -2168,7 +2168,7 @@ impl ProposalData {
         let limited_ballots: BTreeMap<_, _> = ballots
             .iter()
             .filter(|(neuron_id, _)| caller_neurons_set.contains(*neuron_id))
-            .map(|(neuron_id, ballot)| (neuron_id.clone(), ballot.clone()))
+            .map(|(neuron_id, ballot)| (neuron_id.clone(), *ballot))
```

Comment:
Is NeuronId not copy?

Reply:
It was not copy, now it is copy, b/c of prost change (I learned this in response to a similar question)